10up/ElasticPress
# Rename Last Sync in Status Report

### Description of the Change
This PR will rename the Last Sync text in the sub-section of Last Sync present in the Status Report page



Closes #3208 

### How to test the Change

- Go to ElasticPress->Status Report and observe that the Last Sync section will show the date in the sub-section title
- Reset the settings by running `wp elasticpress settings-reset --yes`
- Go to ElasticPress->Status Report and see that the Last Sync section is not visible. 


### Changelog Entry

> Changed - Last Sync subsection title


### Credits

Props @MARQAS @tomioflagos

### Checklist:
